ccmlib/common.py

```diff
@@ -39,6 +40,14 @@
 CONFIG_FILE = "config"
 CCM_CONFIG_DIR = "CCM_CONFIG_DIR"
 
+logger = logging.getLogger('ccm')
+ccm_log_level = os.environ.get('CCM_LOG_LEVEL', None)
```

Comment:
I think all the level control can be handled only in dtest.

and should maybe part of the command line options for when ccm is called directly

Reply:
ccm can also run as a standalone program